IDE/GitISPFClient/sbgzexec/bgzconf.rexx

```diff
@@ -49,6 +49,7 @@
   Perl5_HOME   = '/var/rocket'
   DBB_HOME     = '/var/dbb103'
   ICONV_HOME   = '/bin/iconv'
+  CAINFO       = 'export GIT_SSL_CAINFO=/u/dohertl/Test/cacert.pem'
```

Comment:
@xhouis why are we adding a line with a developer's directory and file?

Reply:
@drbruce-git Good point, we need to change this to
 CAINFO       = 'export GIT_SSL_CAINFO=/var/rocket/cacert.pem'
Do i need to push the fix on same local branch ?

Reply:
@xhouis  yes if you make the change on your local branch and then commit and push to your fork, it should be picked up automatically by the pull request.

Reply:
@drbruce-github Ok done